src/cmd/go/internal/test/test.go

```diff
@@ -1288,7 +1294,7 @@ func (c *runCache) builderRunTest(b *work.Builder, ctx context.Context, a *work.
 			cmd.Stdout.Write([]byte("\n"))
 		}
 		fmt.Fprintf(cmd.Stdout, "ok  \t%s\t%s%s%s\n", a.Package.ImportPath, t, coveragePercentage(out), norun)
-		c.saveOutput(a)
+		c.saveOutput(a, tempCoverProfile)
```

Comment:
This changes to `r.c.saveOutput(a)` in the latest `1.20` changes, triggering a merge conflict.